server.js

```diff
@@ -1,38 +1,100 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-import mongoose from 'mongoose'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
+import express from "express";
+import bodyParser from "body-parser";
+import cors from "cors";
+import mongoose from "mongoose";
+import netflixData from "./data/netflix-titles.json";
+
 //
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
 
-// Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/netflix";
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = Promise;
+
+const Movie = mongoose.model("Movie", {
+  show_id: {
+    type: Number
+  },
+  title: {
+    type: String
+  },
+  director: {
+    type: String
+  },
+  cast: {
+    type: String
+  },
+  Country: {
+    type: String
+  },
+  date_added: {
+    type: Number
+  },
+  release_Year: {
+    type: Number
+  },
+  rating: {
+    type: String
+  },
+  duration: {
+    type: String
+  },
+  listed_In: {
+    type: String
+  },
+  description: {
+    type: String
+  },
+  type: {
+    type: String
+  }
+});
+
+const addMovieToDatabase = () => {
+  netflixData.forEach(movie => {
+    new Movie(movie).save();
+  });
+};
+addMovieToDatabase();
```

Comment:
careful. this will add to the database every time the server starts. And will give you duplicates.